.github/workflows/devnet-deploys.yml

```diff
@@ -83,29 +108,33 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
+          aws-region: eu-west-2
+
+      - name: Deploy mainnet fork
+        if: needs.build.outputs.mainnet_fork_changed == 'true'
+        working-directory: ./iac/mainnet-fork/terraform
+        run: |
+          terraform init -input=false -backend-config="key=${{ env.DEPLOY_TAG }}/mainnet-fork"
+          terraform apply -input=false -auto-approve -replace="aws_ecs_service.aztec_mainnet_fork" -replace="aws_efs_file_system.aztec_mainnet_fork_data_store"
+
+      - name: Wait for mainnet fork deployment
+        if: needs.build.outputs.mainnet_fork_changed == 'true'
+        run: |
+          ./.github/scripts/wait_for_fork.sh ${{ env.DEPLOY_TAG }} ${{ secrets.FORK_API_KEY }}
 
-      - name: Check if L1 contracts need deployment
-        id: check_changes_release
-        uses: actions/github-script@v7
-        with:
-          script: |
-            const { execSync } = require('child_process');
-            const changedFiles = execSync('git diff --name-only ${{ github.event.before }} ${{ github.sha }}').toString().split('\n');
-            const fileChanged = changedFiles.includes('l1-contracts/REDEPLOY');
-            return fileChanged
       - name: Deploy L1 Contracts
-        if: steps.check_changes_release.outputs.result == 'true'
+        if: needs.build.outputs.l1_contracts_changed == 'true' || needs.build.outputs.mainnet_fork_changed == 'true'
         run: |
           docker pull aztecprotocol/cli:${{ env.DEPLOY_TAG }}
-          docker run aztecprotocol/cli:${{ env.DEPLOY_TAG }} \
-            deploy-l1-contracts -p ${{ secrets.SEQ_1_PUBLISHER_PRIVATE_KEY }} \
+          docker run aztecprotocol/cli:${{ env.DEPLOY_TAG }} deploy-l1-contracts \
+            -p ${{ secrets.SEQ_1_PUBLISHER_PRIVATE_KEY }} \
```

Comment:
I was going to comment that `-p` is for exposing a port, when I noticed it was a CLI command of ours, not of docker, heh. WDYT about using long command names in our scripts in the future, to be more descriptive?

Reply:
hmm you mean like using `--private-key` rather than `-p`? I was actually think it should be `-pk` anyway.
Or you mean putting this long command into a script that is then used in here

Reply:
> you mean like using --private-key rather than -p?

Yep, that! Sorry, I meant "long option names", not "long command names"